src/Form/Element/Select.php

```diff
@@ -252,6 +257,41 @@ public function getFetchColumns()
     }
 
     /**
+     * Set Callback for prepare load options Query.
+     *
+     * Example:
+     * <code>
+     * <?php
+     * AdminFormElement::select('column', 'Label')
+     *     ->modelForOptions(MyModel::class)
+     *     ->setLoadOptionsQueryPreparer(function($item, QueryBuilder $query) {
+     *         return $query
+     *             ->where('column', 'value')
+     *             ->were('owner_id', Auth::user()->id)
+     *     });
+     * ?>
+     * </code>
+     *
+     * @param callable $callback The Callback with $item and $options args.
+     * @return $this
+     */
+    public function setLoadOptionsQueryPreparer($callback)
```

Comment:
I can not define the type of 'Callback' for 'Closure', it would be inconsistent if I pass an object of a class whose method '__invoke' was implemented.

```php
class MyCaller {
    public function __invoke($item, QueryBuilder $query) {
        // ....
        return $query;
    }
}

AdminFormElement::select('column', 'Label')
    ->modelForOptions(MyModel::class)
    ->setLoadOptionsQueryPreparer(new MyCaller());
```

The PHP will raise the error:

```
PHP Catchable fatal error:  Argument 2 passed to setLoadOptionsQueryPreparer() must be an instance of Closure, instance of MyCallback given, called in [...]
```